resources/views/pages/admin/link-teacher-student.blade.php

```diff
@@ -0,0 +1,194 @@
+@extends('layout.HUdefault')
+
+@section('title')
+    Admin dashboard
+@stop
+@section('content')
+
+    <div class="container-fluid">
+
+        <h1>{{ __('linking.overzicht') }}</h1>
+        <div class="row">
+
+
+            <div class="col-md-12">
+                <div class="panel panel-default">
+                    <div class="panel-body">
+                        <h3>{{ __('linking.docenten') }}</h3>
+                        <hr/>
+                        <div class="table-responsive">
+                            <table class="table table-striped">
+
+                                <thead>
+                                <tr>
+                                    <th>@sortablelink('firstname', 'First name')</th>
+                                    <th>@sortablelink('lastname', 'Last name')</th>
+                                    <th>@sortablelink('email', 'E-mail')</th>
+                                    <th></th>
+                                </tr>
+                                </thead>
+
+                                <tbody>
+                                <?php
+                                use App\Repository\SearchFilter;use App\Student;
+                                /** @var Student $student */ ?>
+                                @foreach($students as $student)
```

Comment:
You are currently using one array for two purposes: displaying teachers and displaying students, this gives several issues because this array is a paginated collection, i.e. it takes the page parameter from the request and takes a chunk of all students.

- The teachers table is borked because the array is paginated, see image. The paginator code sees an array of many entries, but only a part is shown. But the pages are still rendered as links to navigate to. A solution would be to filter _before_ looping.
![image](https://user-images.githubusercontent.com/4541046/68023670-24959980-fca8-11e9-80fa-75c53f4fb818.png)

- The students you can search for in the link modal are only the students from the same array as above, so if you're on page 1 you only get the first 25 (default pagination chunk) students to link. You would need a